python/ray/tune/trainable/trainable.py

```diff
@@ -65,6 +67,11 @@
 
 SETUP_TIME_THRESHOLD = 10
 
+# File containing dict data returned by user from `Trainable.save_checkpoint`
+_DICT_CHECKPOINT_FILE_NAME = "dict_checkpoint.pkl"
+# Marker file indicating that a checkpoint is a dict checkpoint.
+_DICT_CHECKPOINT_MARKER = ".is_dict_checkpoint"
```

Comment:
Could we check the existence of dict_checkpoint.pkl instead?

Reply:
I was worried about users writing a file named the same thing in their `save_checkpoint`. We could add an underscore at the beginning and most likely this won't cause any problems?

Also think that the marker is ok.

Reply:
Underscore sounds good. This is just a temporary thing anyways right? I want to really get rid of as much marker craft as possible to keep it straightforward.

Reply:
I think we'll need to keep this part of the implementation around even after 2.7. Unless we want to change the `save_checkpoint` API to not accept a dict anymore. I'll remove the marker.